pkg/aggregator/ckey/key.go

```diff
@@ -91,11 +91,11 @@ func (g *KeyGenerator) Generate(name, hostname string, tagsBuf *util.TagsBuilder
 	//   - n > hashSetSize:         sort
 
 	tags := tagsBuf.Get()
+	hashes := tagsBuf.Hashes()
 
 	if len(tags) > hashSetSize {
 		tagsBuf.SortUniq()
-		for _, tag := range tagsBuf.Get() {
-			h := murmur3.StringSum64(tag)
+		for _, h := range tagsBuf.Hashes() {
```

Comment:
```suggestion
		for _, h := range hashes {
```

Reply:
In this case no, as SortUniq would truncate inner slice. I moved local slice copies into the `if` branches to avoid confusion.